extensions/amp-web-push/0.1/test/test-web-push-service.js

```diff
@@ -67,6 +67,22 @@ describes.realWin('web-push-service environment support', {
     });
     expect(webPush.environmentSupportsWebPush()).to.eq(false);
   });
+
+  it('an unsupported environment should prevent initializing', () => {
+    // Cause push to not be supported on this environment
+    Object.defineProperty(env.win, 'PushManager', {
+      enumerable: false,
+      configurable: false,
+      writable: false,
+      value: undefined,
+    });
+    // Should not error out
+    webPush.start().then(() => {
+      throw new Error('must never happen');
```

Comment:
doing 
```
return webPush.start().should.eventually.be.rejectedWith(/Web push is not supported/);
```
would be simpler.

Reply:
@aghassemi Thanks for catching that and the shorter version. Updated the test.